### PR TITLE
[UI] Counter in Performance Profile fixed

### LINKED ISSUE #4479: [UI] Counter in Performance Profile (replicate)
#### Current Behavior
No counter is shown on the selection of multiple selections from bulk delete

#### Desired Behavior
A counter in prompt of delete modal while selecting more than one row.
Reference : [PR](https://github.com/meshery/meshery/pull/4430)

#### Environment

- Browser: Chrome Safari Firefox
- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by ruhi09**
Signed-off-by: ruhi09 <rim.sen5613@gmail.com>

**Description**
Count displayed in Performance when content is deleted in bulk. Handles both the case (count == 1 and count > 1).
Capture
Capture2

This PR fixes #4479 

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-devel
oper-certificate-of-origin)**
- [x] Yes, I signed my commits

### CODE REVIEW

File: ui/components/MesheryPerformance/PerformanceProfiles.js [hunk 1]
```diff
@@ -116,11 +116,9 @@ function PerformanceProfile({ updateProgress, enqueueSnackbar, closeSnackbar })
     });
   }
 
-  async function showModal() {
-    let response = await modalRef.current.show({
-      title : "Delete Performance Profile?",
-
-      subtitle : "Are you sure you want to delete this performance profile?",
+  async function showModal(count) {
+    let response = await modalRef.current.show({ title : `Delete ${count ? count : ""} Application${count > 1 ? "s" : '' }?`,
```
Comment (Chadha93) [inline ui/components/MesheryPerformance/PerformanceProfiles.js:120]: @ruhi09 please change Applications to performance profile in title

File: ui/components/MesheryPerformance/PerformanceProfiles.js [hunk 2]
```diff
@@ -116,11 +116,9 @@ function PerformanceProfile({ updateProgress, enqueueSnackbar, closeSnackbar })
...
   }
 
-  async function showModal() {
-    let response = await modalRef.current.show({
-      title : "Delete Performance Profile?",
-
-      subtitle : "Are you sure you want to delete this performance profile?",
+  async function showModal(count) {
+    let response = await modalRef.current.show({ title : `Delete ${count ? count : ""} Application${count > 1 ? "s" : '' }?`,
+      subtitle : `Are you sure you want to delete ${count > 1 ? "these" : 'this' } ${count ? count : ""} application${count > 1 ? "s" : '' }?`,
```
Comment (Chadha93) [inline ui/components/MesheryPerformance/PerformanceProfiles.js:121]: here also applications to performance profiles

**Review by Chadha93 — CHANGES_REQUESTED** [review]
Added few comments, please do update that, then we are good to go.

**Comment by Chadha93** [comment]
+ update the new screenshots of the performance profile in the description, if you would.

**Review by warunicorn19 — APPROVED** [review]
LGTM, thanks @ruhi09

**Review by Chadha93 — APPROVED** [review]
Nice work, 
Thanks @ruhi09

**Review by sayantan1413 — APPROVED** [review]

**Review by leecalcote — APPROVED** [review]

**Pull request merged**